doc/source/whatsnew/v0.24.0.txt

```diff
@@ -16,7 +16,9 @@ Other Enhancements
 - :func:`Series.mode` and :func:`DataFrame.mode` now support the ``dropna`` parameter which can be used to specify whether NaN/NaT values should be considered (:issue:`17534`)
 - :func:`to_csv` now supports ``compression`` keyword when a file handle is passed. (:issue:`21227`)
 - :meth:`Index.droplevel` is now implemented also for flat indexes, for compatibility with MultiIndex (:issue:`21115`)
-
+- Allow elementwise coloring in ``style.background_gradient`` with ``axis=None`` (:issue:`15204`)
```

Comment:
use ``:class:`pandas.io.formats.style.Styler` ``

Reply:
or even better, I'll move it alongside with the changelog entry for PR #21263